options.js

```diff
@@ -88,88 +94,123 @@ function updateShortcutInputText(inputId, keyCode) {
   document.getElementById(inputId).keyCode = keyCode;
 }
 
+function updateCustomShortcutInputText(inputItem, keyCode) {
+  inputItem.value = keyCodeAliases[keyCode] || String.fromCharCode(keyCode);
+  inputItem.keyCode = keyCode;
+}
+
+function add_shortcut() {
+  var html = '<select class="customDo">' +
+    '<option value="slower">Decrease speed</option>' +
+    '<option value="faster">Increase speed</option>' +
+    '<option value="rewind">Rewind</option>' +
+    '<option value="advance">Advance</option>' +
+    // '<option value="reset">Reset speed</option>' +
```

Comment:
Any particular reason why this one is disabled?

Reply:
Given that we provide this option today, I think we should keep. I'm sure folks will complain otherwise :)